es-vue-base/src/lib-utils/nav-bar-global-content.js

```diff
@@ -441,17 +443,17 @@ export default (
             items: [
                 {
                     name: 'Solar for businesses',
-                    link: `${NEWS_DOMAIN}/commercial-solar-benefits-for-business/`,
+                    link: `${ES_DOMAIN}/business-solutions/commercial-solar-benefits/`,
                     newTab: true,
                 },
                 {
                     name: 'Solar for nonprofit organizations',
-                    link: `${NEWS_DOMAIN}/solar-nonprofit-benefits-financing/`,
+                    link: `${ES_DOMAIN}/business-solutions/solar-nonprofit-benefits-financing/`,
                     newTab: true,
                 },
                 {
                     name: 'Energy storage for businesses',
-                    link: `${NEWS_DOMAIN}/energy-storage-for-businesses/`,
+                    link: `${ES_DOMAIN}/business-solutions/energy-storage-for-businesses/`,
                     newTab: true,
```

Comment:
Anything that was pointing to `NEWS_DOMAIN` before and has `newTab` set to `true` should remove the `newTab: true` line, effectively setting it to `false`, now that everything is on the same domain (and, importantly, has the same navigation).